src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/IVsTargetFrameworkInfo.cs

```diff
@@ -19,6 +19,12 @@ public interface IVsTargetFrameworkInfo
         string TargetFrameworkMoniker { get; }
 
         /// <summary>
+        /// Collection of project level properties evaluated per each Target Framework,
+        /// e.g. PackageTargetFallback.
+        /// </summary>
+        IVsReferenceProperties Properties { get; }
```

Comment:
Can we new types for this, `IVsProjectProperties` and `IVsProjectProperty`? This is clearer to what is being contained here and allows the collections to evolve separately in the future.